app/controllers/api_controller/providers.rb

```diff
@@ -5,9 +5,14 @@ module Providers
     CREDENTIALS_ATTR  = "credentials"
     AUTH_TYPE_ATTR    = "auth_type"
     DEFAULT_AUTH_TYPE = "default"
-    ENDPOINT_ATTRS    = %w(hostname ipaddress port)
+    ENDPOINT_ATTRS    = %w(hostname ipaddress port connection_configurations_array).freeze
```

Comment:
what do the 2 values look like in the response ?

```
GET /api/providers/:id?attributes=connection_configurations,connection_configurations_array
```

I'm not crazy about the name connection_configurations_array, if we need it, maybe something like connection_configuration_definitions or simply connection_definitions.  @imtayadeway thoughts ?